nemo/collections/nlp/models/language_modeling/megatron_gpt_sft_model.py

```diff
@@ -180,6 +184,15 @@ def setup(self, stage=None):
         if hasattr(self.cfg.data, 'test_ds'):
             self._test_dl = self.setup_eval_dataloader(self._test_ds, self.cfg.data.test_ds)
 
+        # If setting val and test dataloaders via this method, create self.validation_step_outputs and self.test_step_outputs
+        if type(self._validation_dl) == list and len(self._validation_dl) > 1:
+            for _ in range(len(self._validation_dl)):
+                self.validation_step_outputs_sft.append([])
+
+        if type(self._test_dl) == list and len(self._test_dl) > 1:
+            for _ in range(len(self._test_dl)):
+                self.test_step_outputs.append([])
```

Comment:
I thought multiple dataloaders wasn't supported yet by PTL 2.0 when using `dataloader_iter` ?

Reply:
Yes that's right. Added it more for futuristic purpose. Let me know if we want to remove it.

Reply:
Discussed offline and fixed.